Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Matomo/CommunityPlugin IPtoCompany](https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/matomo/-/communityplugin-iptocompany/horizontal-auto.svg)

Findus23 commented:
See https://github.com/Romain/Matomo-IP-to-Company/issues/11

I have created the component here: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/

Please don't merge this PR yet, in the next 24 hours it will be updated to add all other languages and afterwards you should always recieve a PR with any changes that you can e.g. merge before releases.

The only thing you need to do is this:
For Weblate to be notified anytime the source strings change, you need to set up a Webhook. For this go into the settings of this repository and create a new webhook with https://hosted.weblate.org/hooks/github/ as the Payload URL. You can leave all other settings at the default.

Findus23 commented:
And I have replaced `...` with `…` to get rid of one notification in weblate

weblate committed:
Translated using Weblate (English)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/en/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lukas Winkler <weblate@lw1.at>

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/es/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lukas Winkler <weblate@lw1.at>

weblate committed:
Translated using Weblate (French)

Currently translated at 95.2% (20 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/fr/

[ci skip]

Translated using Weblate (French)

Currently translated at 95.2% (20 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/fr/

[ci skip]

Co-authored-by: Cyrille Sanson <cyrille@100son.net>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Added translation using Weblate (Irish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/sv/

[ci skip]

Translated using Weblate (Swedish)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/sv/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lukas Winkler <weblate@lw1.at>

weblate committed:
Translated using Weblate (German)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/de/

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Stefan <stefan@matomo.org>

weblate committed:
Added translation using Weblate (Telugu)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Lithuanian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Latvian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/tr/

[ci skip]

Added translation using Weblate (Turkish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kaya Zeren <kayazeren@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Thai)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Galician)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Danish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Basque)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/pt_BR/

[ci skip]

Added translation using Weblate (Portuguese (Brazil))

[ci skip]

Co-authored-by: Claudio Filho F Filho <filhocf@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Tamil)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/el/

[ci skip]

Added translation using Weblate (Greek)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Vasilis Lourdas <dev@lourdas.eu>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Estonian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Czech)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/ca/

[ci skip]

Added translation using Weblate (Catalan)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: raf <rafroset@gmail.com>

weblate committed:
Added translation using Weblate (Japanese)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Bosnian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Russian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Romanian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Portuguese)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Belarusian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Icelandic)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Ukrainian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Vietnamese)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/bg/

[ci skip]

Translated using Weblate (Bulgarian)

Currently translated at 23.8% (5 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/bg/

[ci skip]

Added translation using Weblate (Bulgarian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Ministry of Electronic Governance - Bulgaria <dvdimov@e-gov.bg>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Croatian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Amharic)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hindi)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hebrew)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Polish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hungarian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Italian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Arabic)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/id/

[ci skip]

Translated using Weblate (Indonesian)

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/id/

[ci skip]

Added translation using Weblate (Indonesian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Taufik Adi Wicaksono <taufikadi.wicaksono@tutamail.com>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Dutch)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Persian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Norwegian Nynorsk)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Korean)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Albanian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Slovenian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Finnish)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Serbian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Georgian)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Slovak)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Chinese (Simplified))

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (21 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/zh_Hant/

[ci skip]

Added translation using Weblate (Chinese (Traditional))

[ci skip]

Co-authored-by: HO Gin Wang <hoginwang1996@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 85.7% (18 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/nb_NO/

[ci skip]

Translated using Weblate (Norwegian Bokmål)

Currently translated at 80.9% (17 of 21 strings)

Translation: Matomo/CommunityPlugin IPtoCompany
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-iptocompany/nb_NO/

[ci skip]

Added translation using Weblate (Norwegian Bokmål)

[ci skip]

Co-authored-by: Allan Nordhøy <epost@anotheragency.no>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Welsh)

[ci skip]

Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Tagalog)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Spanish (Argentina))

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Urdu)

[ci skip]

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

Romain commented:
Thanks a lot @Findus23 !

I just setup the @weblate webhook. I'll merge this PR.
